app/helpers/repository_helper.rb

```diff
@@ -105,10 +105,12 @@ def remove_files(files, user, repo, path: '/', txn: nil, keep_folder: true)
     end
 
     current_path = Pathname.new path
-
     current_revision = repo.get_latest_revision.revision_identifier
 
     files.each do |file_path|
+      relative_path = Pathname.new(file_path)
```

Comment:
Okay, this is a good start but it would be better if you are able to use the variables that are *already* defined in this loop (see `subdir_path`, `basename`, `file_path` below).